src/aof.c

```diff
@@ -1098,6 +1112,7 @@ int loadSingleAppendOnlyFile(char *filename) {
     long loops = 0;
     off_t valid_up_to = 0; /* Offset of latest well-formed command loaded. */
     off_t valid_before_multi = 0; /* Offset before MULTI command loaded. */
+    off_t base_size = 0, progress_size = 0;
```

Comment:
is `base_size` better named `last_progress_report_size`?
i think "base" is not a good word here (since it keeps changing).